### PR TITLE
Story screen + messagePlayer "type" parameter + ccmds rewrite

### PR DESCRIPTION
**Pull request by SheridanR**
Story screen uses JSON files to define sequences
messagePlayer functions include new "type" parameter
console commands now a variable type, rewrote consolecommand.cpp
change minimap colors
fix player bobbing while using console

Signed-off-by: SheridanR <sheridan.rathbun@gmail.com>

### CODE REVIEW

**Review by WALLOFJUSTICE — CHANGES_REQUESTED** [review]

File: src/menu.cpp [hunk 1]
```diff
@@ -14113,167 +14113,7 @@ void buttonMiscTab(button_t* my)
...
-		net_handler->toggleMultithreading(settings_disableMultithreadedSteamNetworking);
-	}
-	disableMultithreadedSteamNetworking = settings_disableMultithreadedSteamNetworking;
-	disableFPSLimitOnNetworkMessages = settings_disableFPSLimitOnNetworkMessages;
-#ifdef USE_EOS
-	if ( LobbyHandler.settings_crossplayEnabled )
-	{
-		if ( !LobbyHandler.crossplayEnabled )
-		{
-			LobbyHandler.settings_crossplayEnabled = false;
```
[the remark this answers is not in the document]
Comment (SheridanR) [reply src/menu.cpp:14263]: I suppose not, I'd rather not add it back though so it's best just to remember where it was when I canned it

[the remark this answers is not in the document]
Comment (WALLOFJUSTICE) [reply src/menu.cpp:14263]: Can you add the equivalent of this back in to the new settings?

File: src/actplayer.cpp [hunk 2]
```diff
@@ -2458,8 +2460,8 @@ void actPlayer(Entity* my)
 					if ( PLAYER_ALIVETIME == 300 )
 					{
 						playSound(185, 128);
-						messageLocalPlayersColor(color, language[537]);
-						messageLocalPlayersColor(color, language[89]);
+						messageLocalPlayersColor(color, MESSAGE_WORLD, language[537]);
```
[the remark this answers is not in the document]
Comment (SheridanR) [reply src/actplayer.cpp:2463]: It was defined the other way, but then I accidentally put all the args in the wrong order 🤦‍♂️ easier to just switch the parameters as a fix.

File: src/menu.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (SheridanR) [reply src/menu.cpp:14263]: done

**Pull request merged**